app-office/kmymoney/kmymoney-5.1.3.recipe

```diff
@@ -0,0 +1,195 @@
+SUMMARY="Personal finance manager based on KDE Frameworks"
+DESCRIPTION="KMyMoney is a cross-platform personal finance manager build on KDE frameworks \
+technologies for your desktop and notebook environment. It enables users to keep careful track of \
+their personal finances by providing a broad array of financial features and tools."
+HOMEPAGE="https://kmymoney.org/"
+COPYRIGHT="2010-2023 KDE Organisation"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/kmymoney/$portVersion/src/kmymoney-$portVersion.tar.xz"
+CHECKSUM_SHA256="3938b8078b7391ba32e12bb4239762fae134683a0c2ec1a75105c302ca3e5e3f"
+PATCHES="kmymoney-$portVersion.patchset"
+ADDITIONAL_FILES="kmymoney.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="!x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	kmymoney$secondaryArchSuffix = $portVersion
+	app:KMyMoney$secondaryArchSuffix = $portVersion
```

Comment:
and once again: `$appsDir` isn't architecture-specific, IIRC, so:
```suggestion
	app:KMyMoney = $portVersion
```
(sorry for being pedantic ...)

Reply:
No problem, will fix :)